Makefile

```
@@ -154,6 +142,8 @@ antctl-release:
 tidy:
 	@rm -f go.sum
 	@$(GO) mod tidy
+	@rm -f plugins/octant/go.sum
+	@cd plugins/octant; $(GO) mod tidy
```

```suggestion
	@cd plugins/octant && $(GO) mod tidy
```
"&&" is preferred as it ensures it enters the directory before executing the following command.

---

Done.